tests/solids/LiH_solid_1x1x1_pp/CMakeLists.txt

```diff
@@ -60,9 +60,9 @@ IF(ENABLE_SOA)
                     0 LIH_X_SCALARS # VMC
                     )
 ELSE()
-  MESSAGE("Skipping LiH_solid_1x1x1_hybridrep_pp-x-vmc_hf_noj test because SOA is not enabled (ENABLE_SOA=0)")
+  MESSAGE("Skipping LiH_solid_1x1x1_hybridrep_pp-x-vmc_hf_noj test because SOA and/or GPU is not enabled (ENABLE_SOA=0 or QMC_CUDA=0)")
```

Comment:
The logic in the message seems not consistent with the if statement.

Reply:
The logic is fine.

Reply:
For example, when I build SoA without GPU. ENABLE_SOA=1 and QMC_CUDA=0. This sounds like the case skipping the test as the message described.

Reply:
Got it, will update.